src/puppetlabs/puppetdb/cli/import.clj

```diff
@@ -14,7 +14,7 @@
             [puppetlabs.puppetdb.schema :refer [defn-validated]]
             [puppetlabs.puppetdb.utils :as utils
              :refer [base-url-schema export-root-dir]]
-            [puppetlabs.kitchensink.core :refer [cli!]]
+            [puppetlabs.kitchensink.core :as kitchensink :refer [cli!]]
```

Comment:
@ajroetker not all that important, but as long as I was changing it, I might pick something even shorter, like sink.